benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -219,7 +219,7 @@ struct ArborX::AccessTraits<NearestNeighborsSearches<DeviceType>,
 namespace bpo = boost::program_options;
 
 template <class NO>
-int main_(std::vector<std::string> const &args, const MPI_Comm comm)
+int main_(std::vector<std::string> const &args, MPI_Comm const comm)
```

Comment:
This and some other similar changes are related to using a newer clang-format version but I didn't bother to separate since I don't have `clang-format-14` anymore.